docs/source/contents/4_draft_mapping.rst

```diff
@@ -1662,6 +1662,407 @@ opaque:hostItem
+
+Justification
+^^^^^^^^^^^^^
+Translating these to a relative URIs would require significant effort, and the value added may be trivial at this point.
+
+Xpath
+^^^^^
+
+`mods:location/mods:physicalLocation`
+
+Decision
+^^^^^^^^
+In cases that the physicalLocation are non-UTK and only a string is provided, we will only use the string literal.
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>Blount County Public Library</physicalLocation>
+        <holdingExternal>
+            <holding xsi:schemaLocation="info:ofi/fmt:xml:xsd:iso20775 http://www.loc.gov/standards/iso20775/N130_ISOholdings_v6_1.xsd">
+                <physicalAddress>
+                    <text>City: Maryville</text>
+                    <text>County: Blount County</text>
+                    <text>State: Tennessee</text>
+                </physicalAddress>
+            </holding>
+        </holdingExternal>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1>
+        relators:rps "Blount County Public Library" .
+
+physicalLocation with shelfLocator (UTK)
+----------------------------------------
+
+Use Case
+^^^^^^^^
+In many cases, some of our collection items will have shelfLocator information. This information may not currently be accurate and can found via Special Collections’ finding aids.
+
+Justification
+^^^^^^^^^^^^^
+Because our records in MODS records may not be accurate and this information is located elsewhere, and perhaps more accurate, we will drop this information when shelfLocator is used in conjunction with our repositories.
+
+Xpath
+^^^^^
+
+`mods:location/mods:shelfLocator`
+
+Decision
+^^^^^^^^
+We will drop `mods:shelfLocator` data when present for UT Knoxville records.
+
+`Example record from scopes:1258 <https://digital.lib.utk.edu/collections/islandora/object/scopes:1258/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation valueURI="http://id.loc.gov/authorities/names/no2014027633">University of Tennessee, Knoxville. Special Collections</physicalLocation>
+        <shelfLocator>Box 5, Folder 8</shelfLocator>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1>
+        relators:rps <http://id.loc.gov/authorities/names/no2014027633> .
+
+physicalLocation with shelfLocator (non-UTK)
+--------------------------------------------
+
+Use Case
+^^^^^^^^
+Instances where non-UTK held items have shelfLocator information.
+
+Justification
+^^^^^^^^^^^^^
+While, we do not not know if this shelfLocator information is accurate, we will opt to retain it going forward as a string and map to skos:note. Samvera does note some possible future availability of opaque:locationShelfLocator, however this predicate does not exist yet.
+
+Xpath
+^^^^^
+
+`mods:location/mods:shelfLocator` OR other variations...
```

Comment:
We made the decision a while ago to use inline code for xpath sections.  Switch to: 

```
:code:`mods:location/mods:shelfLocator`
```

Also, what is meant by "other variations?" Are there other variations that this xpath selector wouldn't capture that we need to address here?